schemas/vnsutils.json

```diff
@@ -181,42 +181,49 @@
                 "sectionEnd": "vCenter Parameters",
                 "propertyOrder": 190
             },
+            "vcenter_vmfolder": {
```

Comment:
This new item is part of the vCenter infrastructure section in the schema. Line 181, that defines the end of the section, needs to me moved to this entry so that all vCenter params are properly sectioned.